Inline Assembly "nop" instruction causes NullPointerException

Testcase:

```llvm
target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"

define i1 @main() {
  call void asm sideeffect "nop", ""()
  ret i1 0
}
```

```lli``` works as expected, but running this snippet with ```mx lli``` causes the following stacktrace:

```
Exception in thread "main" org.graalvm.polyglot.PolyglotException: java.lang.RuntimeException: java.io.IOException: Error while trying to parse /home/thomas/JKU/java-llvm-ir-builder-dev/sulong/cache/tests/irbuilder/binaryI1/test_i1_and_(false-true).bc
	at com.oracle.truffle.llvm.asm.amd64.AsmFactory.parseArguments(AsmFactory.java:326)
	at com.oracle.truffle.llvm.asm.amd64.AsmFactory.<init>(AsmFactory.java:278)
	at com.oracle.truffle.llvm.asm.amd64.Parser.<init>(Parser.java:71)
	at com.oracle.truffle.llvm.parser.factories.BasicNodeFactory.createInlineAssemblerExpression(BasicNodeFactory.java:1065)
	at com.oracle.truffle.llvm.parser.LLVMBitcodeInstructionVisitor.createInlineAssemblerNode(LLVMBitcodeInstructionVisitor.java:871)
	at com.oracle.truffle.llvm.parser.LLVMBitcodeInstructionVisitor.visit(LLVMBitcodeInstructionVisitor.java:398)
	at com.oracle.truffle.llvm.parser.model.symbols.instructions.VoidCallInstruction.accept(VoidCallInstruction.java:61)
	at com.oracle.truffle.llvm.parser.LLVMBitcodeFunctionVisitor.visit(LLVMBitcodeFunctionVisitor.java:93)
	at com.oracle.truffle.llvm.parser.model.functions.FunctionDefinition.accept(FunctionDefinition.java:118)
	at com.oracle.truffle.llvm.parser.LazyToTruffleConverterImpl.convert(LazyToTruffleConverterImpl.java:96)
	at com.oracle.truffle.llvm.runtime.LLVMFunctionDescriptor$LazyLLVMIRFunction.resolve(LLVMFunctionDescriptor.java:157)
	at com.oracle.truffle.llvm.runtime.LLVMFunctionDescriptor.getLLVMIRFunction(LLVMFunctionDescriptor.java:337)
	at com.oracle.truffle.llvm.parser.LLVMParserRuntime.parse(LLVMParserRuntime.java:133)
	at com.oracle.truffle.llvm.Runner.parseBitcodeFile(Runner.java:208)
	at com.oracle.truffle.llvm.Runner.parse(Runner.java:123)
	at com.oracle.truffle.llvm.Sulong.parse(Sulong.java:92)
	at com.oracle.truffle.api.TruffleLanguage$ParsingRequest.parse(TruffleLanguage.java:515)
	at com.oracle.truffle.api.TruffleLanguage.parse(TruffleLanguage.java:868)
	at com.oracle.truffle.api.TruffleLanguage$LanguageImpl.parse(TruffleLanguage.java:1604)
	at com.oracle.truffle.api.vm.PolyglotLanguageContext.parseCached(PolyglotLanguageContext.java:86)
	at com.oracle.truffle.api.vm.PolyglotContextImpl.eval(PolyglotContextImpl.java:592)
	at org.graalvm.polyglot.Context.eval(Context.java:148)
	at com.oracle.truffle.llvm.Sulong.executeMain(Sulong.java:142)
	at com.oracle.truffle.llvm.Sulong.main(Sulong.java:134)
Original Internal Error: 
java.lang.RuntimeException: java.io.IOException: Error while trying to parse /home/thomas/JKU/java-llvm-ir-builder-dev/sulong/cache/tests/irbuilder/binaryI1/test_i1_and_(false-true).bc
	at com.oracle.truffle.api.TruffleLanguage.parse(TruffleLanguage.java:872)
	at com.oracle.truffle.api.TruffleLanguage$LanguageImpl.parse(TruffleLanguage.java:1604)
	at com.oracle.truffle.api.vm.PolyglotLanguageContext.parseCached(PolyglotLanguageContext.java:86)
	at com.oracle.truffle.api.vm.PolyglotContextImpl.eval(PolyglotContextImpl.java:592)
	at org.graalvm.polyglot.Context.eval(Context.java:148)
	at com.oracle.truffle.llvm.Sulong.executeMain(Sulong.java:142)
	at com.oracle.truffle.llvm.Sulong.main(Sulong.java:134)
Caused by: java.io.IOException: Error while trying to parse /home/thomas/JKU/java-llvm-ir-builder-dev/sulong/cache/tests/irbuilder/binaryI1/test_i1_and_(false-true).bc
	at com.oracle.truffle.llvm.Runner.parse(Runner.java:133)
	at com.oracle.truffle.llvm.Sulong.parse(Sulong.java:92)
	at com.oracle.truffle.api.TruffleLanguage$ParsingRequest.parse(TruffleLanguage.java:515)
	at com.oracle.truffle.api.TruffleLanguage.parse(TruffleLanguage.java:868)
	... 6 more
Caused by: java.lang.NullPointerException
	at com.oracle.truffle.llvm.asm.amd64.AsmFactory.parseArguments(AsmFactory.java:326)
	at com.oracle.truffle.llvm.asm.amd64.AsmFactory.<init>(AsmFactory.java:278)
	at com.oracle.truffle.llvm.asm.amd64.Parser.<init>(Parser.java:71)
	at com.oracle.truffle.llvm.parser.factories.BasicNodeFactory.createInlineAssemblerExpression(BasicNodeFactory.java:1065)
	at com.oracle.truffle.llvm.parser.LLVMBitcodeInstructionVisitor.createInlineAssemblerNode(LLVMBitcodeInstructionVisitor.java:871)
	at com.oracle.truffle.llvm.parser.LLVMBitcodeInstructionVisitor.visit(LLVMBitcodeInstructionVisitor.java:398)
	at com.oracle.truffle.llvm.parser.model.symbols.instructions.VoidCallInstruction.accept(VoidCallInstruction.java:61)
	at com.oracle.truffle.llvm.parser.LLVMBitcodeFunctionVisitor.visit(LLVMBitcodeFunctionVisitor.java:93)
	at com.oracle.truffle.llvm.parser.model.functions.FunctionDefinition.accept(FunctionDefinition.java:118)
	at com.oracle.truffle.llvm.parser.LazyToTruffleConverterImpl.convert(LazyToTruffleConverterImpl.java:96)
	at com.oracle.truffle.llvm.runtime.LLVMFunctionDescriptor$LazyLLVMIRFunction.resolve(LLVMFunctionDescriptor.java:157)
	at com.oracle.truffle.llvm.runtime.LLVMFunctionDescriptor.getLLVMIRFunction(LLVMFunctionDescriptor.java:337)
	at com.oracle.truffle.llvm.parser.LLVMParserRuntime.parse(LLVMParserRuntime.java:133)
	at com.oracle.truffle.llvm.Runner.parseBitcodeFile(Runner.java:208)
	at com.oracle.truffle.llvm.Runner.parse(Runner.java:123)
	... 9 more
```

--- a/projects/com.oracle.truffle.llvm.asm.amd64/src/com/oracle/truffle/llvm/asm/amd64/AsmFactory.java
+++ b/projects/com.oracle.truffle.llvm.asm.amd64/src/com/oracle/truffle/llvm/asm/amd64/AsmFactory.java
@@ -289,6 +289,9 @@ class AsmFactory {
         int outIndex = 0;
 
         for (String token : tokens) {
+            if (token.isEmpty()) {
+                continue;
+            }
             boolean isTilde = false;
             boolean isInput = true;
             boolean isOutput = false;

